airplane_2.py

```diff
@@ -0,0 +1,25 @@
+class airplane:
+    '''airplane class: 
+    Attributes: location, health, velocity, acceleration
+    Functions: set acceleration, velocity and location. 
+    AirPlane object is controlled by set acceleration function'''
+    def __init__(self, spawn_location:[float,float,float], health, initial_velocity = [float,float,float],
+        initial_acceleration = [float,float,float]):

```

Comment:
This is not the correct type annotation syntax. Please view: https://docs.python.org/3.7/library/typing.html#module-typing